build.py

```diff
@@ -1768,6 +1785,29 @@ def backend_build(
     cmake_script.blankln()
 
 
+def backend_clone(
+    be,
+    cmake_script,
```

Comment:
cmake_script -> clone_script? 
Just for this scope?

Reply:
Sure, done.